integrations/plugins/plugin-gamma/plugin-gamma.php

```diff
@@ -40,6 +42,9 @@
 <div class="plugin-gamma-pre-content" style="border: 1px solid grey;">
   <h2>Plugin Gamma</h2>
   <p>Expected by plugin-gamma (v4.7.0 icon name): "fa fa-bathtub": <i class="fa fa-bathtub"></i></p>
+  <p class="group-icon">Also expected: before this paragraph, pseudo-element (v4.7.0 icon name group should become v5 users)</p>
+  <p class="facebook-icon">Also expected: before this paragraph, pseudo-element (v4.7.0 icon name facebook should become v5 facebook-f)</p>
```

Comment:
This icon doesn't work in this integration test. Square box. I'm sure it's because it's not enough to map the `font-family` of "FontAwesome" to use `fa-solid-900` font files. The unicode for this icon would not be found in those font files in v5, but rather in the `fa-brands` font file. What should be doing to accommodate those brand icons?